tests/serverpod_test_server/test/serialization_test.dart

```diff
@@ -182,39 +201,56 @@ void main() {
-        aListWithNullableObjects: [SimpleData(num: 10), null],
-        aNullableListWithNullableObjects: [SimpleData(num: 10), null],
-        aDateTimeList: [DateTime.utc(1976), DateTime.utc(1977)],
-        aNullableDateTimeList: [DateTime.utc(1976), DateTime.utc(1977)],
-        aListWithNullableDateTimes: [DateTime.utc(1976), null],
-        aNullableListWithNullableDateTimes: [DateTime.utc(1976), null],
-        aByteDataList: [createByteData(), createByteData()],
-        aNullableByteDataList: [createByteData(), createByteData()],
-        aListWithNullableByteDatas: [createByteData(), null],
-        aNullableListWithNullableByteDatas: [createByteData(), null],
-        anIntMap: {'0': 0, '1': 1, '2': 2},
-        aMapWithNullableInts: {'0': 0, '1': null, '2': 2},
-      );
+          anInt: 42,
+          aNullableInt: 42,
+          aDouble: 42.42,
+          aNullableDouble: 42.42,
+          aBool: true,
+          aNullableBool: true,
+          aString: 'foo',
+          aNullableString: 'foo',
+          aDateTime: DateTime.utc(1976),
+          aNullableDateTime: DateTime.utc(1976),
+          aByteData: createByteData(),
+          aNullableByteData: createByteData(),
+          anObject: SimpleData(num: 42),
+          aNullableObject: SimpleData(num: 42),
+          anIntList: [10, 20],
+          aNullableIntList: [10, 20],
+          aListWithNullableInts: [10, null],
+          aNullableListWithNullableInts: [10, null],
+          anObjectList: [SimpleData(num: 10), SimpleData(num: 20)],
+          aNullableObjectList: [SimpleData(num: 10), SimpleData(num: 20)],
+          aListWithNullableObjects: [SimpleData(num: 10), null],
+          aNullableListWithNullableObjects: [SimpleData(num: 10), null],
+          aDateTimeList: [DateTime.utc(1976), DateTime.utc(1977)],
+          aNullableDateTimeList: [DateTime.utc(1976), DateTime.utc(1977)],
+          aListWithNullableDateTimes: [DateTime.utc(1976), null],
+          aNullableListWithNullableDateTimes: [DateTime.utc(1976), null],
+          aByteDataList: [createByteData(), createByteData()],
+          aNullableByteDataList: [createByteData(), createByteData()],
+          aListWithNullableByteDatas: [createByteData(), null],
+          aNullableListWithNullableByteDatas: [createByteData(), null],
+          anIntMap: {'0': 0, '1': 1, '2': 2},
+          aMapWithNullableInts: {'0': 0, '1': null, '2': 2},
+          aDurationList: const [
+            Duration(seconds: 1),
+            Duration(minutes: 1),
+          ],
+          aListWithNullableDurations: const [
+            Duration(seconds: 1),
+            null,
+          ],
+          aNullableDurationList: const [
+            Duration(seconds: 1),
+            Duration(minutes: 1),
+          ],
+          aDuration: const Duration(seconds: 1),
+          aNullableDuration: const Duration(seconds: 1),
+          aNullableListWithNullableDurations: [
+            const Duration(seconds: 1),
+            null,
```

Comment:
Please add a trailing comma. The missing comma makes the whole block being changed in this commit, which may introduce conflicts if someone else is adding another type or if this change needs to be reversed in the future.